src/agent/onefuzz-supervisor/src/coordinator.rs

```diff
@@ -310,9 +312,11 @@ impl Coordinator {
             response = request
                 .send_retry_default()
                 .await
-                .context("Coordinator.send after refreshing access token")?;
+                .context("Coordinator.send after refreshing access token");
         };
 
+        let response = response?;
```

Comment:
Might be good to have something like:
```suggestion
        let response = response.context("non-status error after ensuring valid access token")?;
```